src/components/Header/TopNav/Dropdown.js

```diff
@@ -14,6 +14,7 @@ const DropdownContainer = styled(TopNavItem)`
   position: relative;
   cursor: pointer;
   background: transparent;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
```

Comment:
Just removing the tap highlight on these topnav items

https://css-tricks.com/snippets/css/remove-gray-highlight-when-tapping-links-in-mobile-safari/